runtime/util/srphashtable.c

```diff
@@ -158,7 +158,9 @@ srpHashTableNew(
 
 	/*Allocate memory for struct J9SRPHashTable*/
 	srpHashTable = OMRPORT_FROM_J9PORT(PORTLIB)->mem_allocate_memory(OMRPORT_FROM_J9PORT(PORTLIB), sizeof(J9SRPHashTable), tableName, OMRMEM_CATEGORY_VM);
+#ifdef SRPHASHTABLE_DEBUG
```

Comment:
I think it would be better to fix `srpHashTable_printf`:
```
#ifdef SRPHASHTABLE_DEBUG
#define srpHashTable_printf(...) j9tty_printf(__VA_ARGS__)
#else
#define srpHashTable_printf(...)
#endif
```